.github/workflows/clang-format.ignore.yaml

```diff
@@ -0,0 +1,28 @@
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+# SPDX-FileCopyrightText: Copyright 2022 Micron Technology, Inc.
+
+name: ClangFormat
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+    paths-ignore:
+      - "**.c"
+      - "**.h"
+      - "**.h.in"
+      - "!subprojects/*"
+      - .github/workflows/clang-format.yaml
+
+concurrency:
+  group: ${{ github.workflow }}-ignore-${{ github.head_ref }}
+  cancel-in-progress: true
+
+permissions: {}
+
+jobs:
+  clang-format:
+    runs-on: ubuntu-latest
+    steps:
+      - run: |
+          echo "Skipping ${{ github.workflow }}/${{ github.job }}"
```

Comment:
What's the `.ignore.yaml` file for?

Reply:
To satisfy the "required" checks. If you remember I had a conversation with you and @bhaveshvasandani about removing these since they are so annoying.